src/main/java/com/atlassian/jira/cloud/jenkins/config/ConfigManagementLink.java

```diff
@@ -205,7 +238,20 @@ public void doSaveConfiguration(final StaplerRequest req, final StaplerResponse
             doIndex(req, res);
             return;
         }
-        res.sendRedirect("/jenkins/manage/");
+        redirectToManagePage(req, res);
+    }
+
+    public void redirectToManagePage(final StaplerRequest req, final StaplerResponse res) {
+        String jenkinsRootUrl = Jenkins.get().getRootUrl();
+
+        try {
+            if (jenkinsRootUrl != null) {
+                res.sendRedirect(jenkinsRootUrl + "manage/");
+            }
+        } catch (IOException e) {
+            LOGGER.log(Level.SEVERE, "Success redirect failed", e);
```

Comment:
Can we change this? Seeing 'success' in an error message kinda melts my brain a little.

Reply:
hahaha yeah thats fair